content/contribute/contribute.md

```diff
@@ -31,7 +31,7 @@ the Crossplane docs CSS.
 Extract and run Hugo with `hugo server`.
 
 Hugo builds the website and launch a local web server on
-[http://localhost:1313](http://localhost:1313).
+<a href="http://localhost:1313" data-proofer-ignore>http://localhost:1313</a>.
```

Comment:
the "localhost" link needs to use the `data-proofer-ignore` [htmltest flag](https://github.com/wjdp/htmltest#see_no_evil-ignoring-content)